tests/conditional_compilation/conftest.py

```diff
@@ -81,13 +89,19 @@ def pytest_generate_tests(metafunc):
 @pytest.fixture(scope="session")
```

Comment:
Review scopes when fixture should be available/executed. Some fixtures are used only for specific tests